### PR TITLE
chore: add Readme.md file

### PR DESCRIPTION
**Pull request by Nedum84**

### CODE REVIEW

**Review by jhony112 — CHANGES_REQUESTED** [review]
Please add DS_Store to list of git ignore files

File: assets/js/squad.js [hunk 1]
```diff
@@ -56,13 +56,13 @@ jQuery(function ($) {
       } catch (error) {
         res = response;
       }
-      const responseTransactionRef = res.transaction_ref;
+      const respTxnref = res.transaction_ref;
```
Comment (jhony112) [inline assets/js/squad.js]: naming convention is a bit vague, let's use the former or change to something more readable 😀

File: assets/js/squad.js [hunk 2]
```diff
@@ -82,7 +82,7 @@ jQuery(function ($) {
       });
     };
     const channels =
-      payment_options.length === 0 ? ["card", "transfer", "ussd", "bank"] : payment_options;
```
Comment (jhony112) [inline assets/js/squad.js]: Triple operator checks value and type better in direct comparison

**Review by jhony112 — COMMENTED** [review]
We actually need a readMe.txt not .md please check the slack messages I sent a sample

**Review by jhony112 — APPROVED** [review]
ACK : nice !!

@Nedum84 let sadd DS_Store to gitignore and push up

**Comment by Nedum84** [comment]
I have already added `DS_Store` it to gitignore

**Pull request merged**